reference/technical_vision/1.5/architecture.rst

```diff
@@ -0,0 +1,127 @@
+Architecture [WIP]
+==================
+
+Once Oro bundles moved, there is the re-work strategy for v1.5.
+
+The idea is to provide a cleaner and more understandable stack by removing override and "twin bundles".
+
+There is the v1.4 version with Oro bundles moved in src,
+
+```
+src/
+├── Acme
+│   └── Bundle
+│       └── AppBundle
+├── Akeneo
+│   ├── Bundle
+│   │   ├── ClassificationBundle
+│   │   ├── FileStorageBundle
+│   │   └── StorageUtilsBundle
+│   └── Component
+│       ├── Analytics
+│       ├── Classification
+│       ├── Console
+│       ├── FileStorage
+│       └── StorageUtils
+├── Oro
+│   └── Bundle
+│       ├── AsseticBundle           -> -
+│       ├── ConfigBundle            -> -
+│       ├── DataGridBundle          -> -
+│       ├── EntityBundle            -> [Done] - removed (DoctrineOrmMappingsPass has been extracted in Akeneo/StorageUtilsBundle)
```

Comment:
Here we should keep only the bundle name and what we done for it. Like : 
`Oro\Bundle\EntityBundle -> Removed. DoctrineOrmMappingsPass has been extracted in Akeneo/StorageUtilsBundle`